scripts/tests-ui-integration.sh

```diff
@@ -3,46 +3,29 @@
 PUBLISHED_FUEL_PACKAGE_NAME="${PUBLISHED_FUEL_PACKAGE_NAME-"fuels"}"
 PUBLISHED_NPM_TAG="${PUBLISHED_NPM_TAG-"next"}"
 
-# Versions
-FUEL_CORE_VERSION=$(cat ./internal/fuel-core/VERSION)
-FORC_VERSION=$(cat ./internal/forc/VERSION)
-TOOLCHAIN="CI"
-
 # Project
-ROOT_DIR=$(pwd)
-PLAYWRIGHT_DIR="$ROOT_DIR"
-PROJECT_DIR="$ROOT_DIR/test-project"
-
-echo "1. Install toolchains"
-if [ -x "$(command -v fuelup)" ]; then
-  echo "Fuelup exists"
-else
-  echo "Fuelup does not exist - installing 'fuelup'"
-  curl -fsSL https://install.fuel.network/ | sh -s -- --no-modify-path
-  export PATH="${HOME}/.fuelup/bin:${PATH}"
-fi
+REPO_DIR=$(pwd)
+
+echo "1. Add binaries to PATH so that tests can use them"
+PATH="${REPO_DIR}/internal/fuel-core/fuel-core-binaries:${PATH}"
+PATH="${REPO_DIR}/internal/forc/forc-binaries:${PATH}"
 
-fuelup toolchain new $TOOLCHAIN
-fuelup default $TOOLCHAIN
-fuelup component add fuel-core@$FUEL_CORE_VERSION
-fuelup component add forc@$FORC_VERSION
```

Comment:
`fuelup` can only add released versions. Because we're working with an unreleased version, I had to rewrite this logic to take the binaries from our `internal/` packages.